source/extensions/filters/http/ext_proc/processor_state.h

```diff
@@ -105,6 +105,8 @@ class ProcessorState : public Logger::Loggable<Logger::Id::ext_proc> {
   bool bodyReplaced() const { return body_replaced_; }
   bool partialBodyProcessed() const { return partial_body_processed_; }
 
+  void clearRouteCache(envoy::service::ext_proc::v3::CommonResponse common_response);
+
```

Comment:
`const envoy::service::ext_proc::v3::CommonResponse&` to avoid unnecessary copy.